scripts/setup-macos.sh

```diff
@@ -117,14 +131,22 @@ function install_fmt {
   cmake_install -DFMT_TEST=OFF
 }
 
+function openssl_dir {
+  if hash brew 2>/dev/null; then
+    brew --prefix openssl
+  else
+    echo /opt/local/lib/openssl
```

Comment:
It's funny to hardcode this path in the same PR that changes the default install locations to something other than /usr/local. Is there an alternative way to handle this, e.g. can we add a find script? Why does OpenSSL require special handling anyway?

Reply:
This path was hardcoded in the script before my diff, so I didn't change it.  I've no idea why it was done this way.

Reply:
Ah, I see that now below.